### PR TITLE
add_chown_logs_uwsgi

### PR DESCRIPTION
**Pull request by jmuller1984**

### CODE REVIEW

File: fabfile/utils.py [hunk 1]
```diff
@@ -59,6 +59,14 @@
 from fabtools.require.files import temporary_directory
 from fabtools.utils import run_as_root
 
+# Ajoute suite https://github.com/CanalTP/navitia_deployment_conf/pull/365 , permission denied
+
+@task
+@roles('tyr', 'eng', 'ws')
+def chown_logs_uwsgi():
+   for file in env.uwsgi_logfiles:
+     run ('chown www-data: {}'.format(file))
```
Comment (kinnou02) [inline fabfile/utils.py]: This will probably fail on platform where jormungandr and tyr aren't on the same host.

File: fabfile/utils.py [hunk 2]
```diff
@@ -59,6 +59,14 @@
 from fabtools.require.files import temporary_directory
 from fabtools.utils import run_as_root
 
+# Ajoute suite https://github.com/CanalTP/navitia_deployment_conf/pull/365 , permission denied
```
Comment (kinnou02) [inline fabfile/utils.py]: This kind of comment isn't really useful and should be in your commit message.
Also we use english in public repository, and it is recommended to do the same on private repositories

File: fabfile/tasks.py [hunk 3]
```diff
@@ -279,6 +279,7 @@ def upgrade_tyr(up_confs=False, pilot_tyr_beat=True, reload=True):
 @task
 def restart_tyr(tyr_beat=True):
     # restart tyr workers and reload with newer binaries
+    execute(chown_logs_uwsgi)
```
Comment (kinnou02) [inline fabfile/tasks.py]: I would have put it in elsewhere, as this isn't related to the restart of tyr. Maybe in `upgrade_tyr`.

**Comment by pbougue** [comment]
@jmuller1984 Is this ever gonna be corrected and merged?
If there is no response before 2018/08/01, I'll close this PR: no point in letting open something that's not gonna be merged.

**Pull request closed without merging**